### ISSUE TITLE
Make game recognize intro and controller

### ISSUE THREAD

**Issue by raimuse** [justMaku/gpghax#11]
Ok so i already was able to add 5 games all work perfect the system recognizes and uses the Playstation load screen, now my issue is when i add 2 more now they work but controller is not recognized only the sticks to move and no Playstation load at start thanks for the time ill keep working on it.

**Comment by CYB3R187** [comment]
with which you edit the database?

**Comment by raimuse** [comment]
DB browser sq lite

**Comment by raimuse** [comment]
There is something i did with the first 5 that got recognized even the intro of psone but the other two just play without intro and only sticks can move xd

**Comment by raimuse** [comment]
But the database seems correct in everything maybe a file missing somewhere

**Comment by Matteoson** [comment]
Yes you'll have a problem when you set up your 26th game. Off the shelf there is a 25 game limit wall. 

it is mentioned in this thread

https://www.psxhax.com/threads/playstation-classic-is-hacked-ps-classic-game-swapping-videos.6152/page-4#comments

**Comment by raimuse** [comment]
Cool thanks Matteoson i was really hitting a wall then trying to make them work.

I hope someone get’s an easy way to modify the Max count soon
